packages/components/TabList/src/TabList/useTabList.ts

```diff
@@ -1,9 +1,10 @@
 import * as React from 'react';
-import type { View, AccessibilityState, LayoutRectangle } from 'react-native';
+import { type View, type AccessibilityState, type LayoutRectangle, Platform } from 'react-native';
```

Comment:
I think we prefer these to be separate import lines (import type and import) for tree shaking purposes